sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -279,6 +280,7 @@ export class ServiceBusClient {
 // @public
 export interface ServiceBusClientOptions {
     retryOptions?: RetryOptions;
+    userAgentOptions?: UserAgentOptions;
```

Comment:
I'm following the rest of the SDKs here for adding the `userAgentOptions`.

Looks like event-hubs deviated here. 
```ts
export interface EventHubClientOptions {
    retryOptions?: RetryOptions;
    userAgent?: string;
    webSocketOptions?: WebSocketOptions;
}
```

Reply:
I some other discussion thread, were we not avoiding using `core-http` constructs outside of `ServiceBusManagementClient`?

Reply:
Yes.
Are you suggesting 
- `userAgentOptions?: UserAgentOptions;` for SBManagementClient and 
- `userAgent?: string;` for SBClient

?

Reply:
We discussed offline but repeating here.
Totally agree that we should avoid using core-http constructs outside of `ServiceBusManagementClient` so that bundlers can tree-shake it out when just using the AMQP-based code.

That said since this is just using a `type` from `core-http` and doesn't result in the JavaScript code importing `core-http`, I think using the `UserAgentOptions` is a good choice here.